CMP1903M Assessment 1 Base Code/CMP1903M Assessment 1 Base Code/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿//Base code project for CMP1903M Assessment 1
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CMP1903M_Assessment_1_Base_Code
+{
+    class Program
+    {
+        static void Main()
+        {
+            //Local list of integers to hold the first five measurements of the text
+            List<int> parameters = new List<int>();
+
+            //Create 'Input' object
+            Input UserInput = new Input();
+            //Get either manually entered text, or text from a file
+            string option = "";
+            while (true)
+            {
+                Console.WriteLine("1. Do you want to enter the text via the keyboard ?");
+                Console.WriteLine("2. Do you want to read in the text from a file? ");
+                option = Console.ReadLine(); //Gets the users choice
+                if (option == "1" || option == "2")
+                {                                 
+                    break;
+                }
+                Console.WriteLine("Please enter 1 or 2");
+            }
+            
+            
+            List<string> text = new List<string>();
+            if (option == "1") //See if the user chose to enter the sentances manually
+            {
```

Comment:
Sentence mispelled here, consistent throughout so it doesn't affect the code running though.

Reply:
All sentence misspells have been replaced.